test/pipeline_test_runner.cc

```diff
@@ -202,11 +203,15 @@ vector<ast::ParsedFile> index(unique_ptr<core::GlobalState> &gs, absl::Span<core
         }
 
         unique_ptr<parser::Node> nodes;
-        {
+        if (parser == realmain::options::Parser::SORBET) {
+            std::cout << "Parsing with sorbet" << std::endl;
             core::UnfreezeNameTable nameTableAccess(*gs); // enters original strings
 
             auto settings = parser::Parser::Settings{};
             nodes = parser::Parser::run(*gs, file, settings);
+        } else if (parser == realmain::options::Parser::PRISM) {
+            std::cout << "Parsing with prism" << std::endl;
```

Comment:
I'm going to leave these print statements because without some confirmation that we're hitting this code path, there (afaik) no way to tell that we are actually parsing with prism rather than the sorbet parser 😆